src/main/java/com/algorand/algosdk/v2/client/Utils.java

```diff
@@ -28,23 +28,41 @@
     private static Long defaultAppId = 1380011588L;
 
     /**
-     * Construct a DryruynRequest object from a set of transactions
+     * Construct a DryruynRequest object from a set of transactions. 
+     * A DryrunRequest is composed of static balance information.  This function uses the ApplicationCall transaction 
+     * parameters to infer what Application State and Account balance information to query using the client and adds it to the DryrunRequest object. 
+     * If foreign assets are passed, it will also add the creators balance information to the DryrunRequest. 
+     * For additional details refer to {@link #createDryrun(AlgodClient, List, String, Long, Long)}.
```

Comment:
No need for the link if you put the same description in both! Thanks for the extra details.